Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -2169,34 +2169,44 @@ void EditorActionsHandler::UpdateRecentFileActions()
     const QString gameDirPath = gameDir.absolutePath();
 
     m_recentFileActionsCount = 0;
-    int counter = 0;
+
+    int index = 0;
 
     // Update all names
-    for (int index = 0; (index < maxRecentFiles) || (index < recentFilesSize); ++index)
+    for (int counter = 0; counter < maxRecentFiles; ++counter)
```

Comment:
This is super minor but you could do...

```
for (int counter = 0, index = 0; counter < maxRecentFiles; ++counter)
```

to keep the counter local to the loop, but it's not a huge deal